pkg/api/login_oauth.go

```diff
@@ -179,3 +191,22 @@ func OAuthLogin(ctx *middleware.Context) {
 
 	ctx.Redirect(setting.AppSubUrl + "/")
 }
+
+func verifyScopes(got, want []string) error {
+	var missing []string
+
+loop:
```

Comment:
It's used all over the golang itself. Sure if it's an issue I can add refactor it somehow.